src/Microsoft.AspNet.OData.Shared/Builder/PropertyConfiguration.cs

```diff
@@ -157,6 +164,29 @@ public bool Unsortable
         /// </summary>
         public QueryConfiguration QueryConfiguration { get; set; }
 
+        /// <summary>
+        /// Adds TSubClass to the list of derived type constraints.
+        /// </summary>
+        internal PropertyConfiguration AddDerivedTypeConstraintImpl(Type t)
+        {
+            if (RelatedClrType == null)
+            {
+                throw Error.InvalidOperation("No CLR Type specified for this configuration.");
```

Comment:
>No CLR Type specified for this configuration. [](start = 46, length = 45)

add the errors into resource manage

Reply:
same for other

---
In reply to: [362588158](https://github.com/OData/WebApi/pull/1977#discussion_r362588158) [](ancestors = 362588158)